Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyButton.cs

```diff
@@ -36,7 +36,7 @@ public PropertyButton (IHostResourceProvider hostResources)
 
 			this.hostResources = hostResources;
 
-			AccessibilityTitle = Properties.Resources.AccessibilityPropertiesButton;
+			AccessibilityTitle = string.Format (Properties.Resources.AccessibilityPropertiesButton, ViewModel.Property.Name);
```

Comment:
`ViewModel` can't be set in the ctor, so this will always throw an `NRE`. Additionally, it wouldn't update when the button is re-used with a different property.

Reply:
Indeed!